scripts/build.sh

```diff
@@ -40,7 +40,7 @@ mkdir -p bin/
 echo "==> Building..."
 ${GO_CMD} build \
     -gcflags "${GCFLAGS}" \
-    -ldflags "${LD_FLAGS} -X github.com/hashicorp/vault/sdk/version.GitCommit='${GIT_COMMIT}${GIT_DIRTY}' -X github.com/hashicorp/vault/sdk/version.BuildDate=${BUILD_DATE}" \
+    -ldflags "${LD_FLAGS} -X github.com/hashicorp/vault/version.GitCommit='${GIT_COMMIT}${GIT_DIRTY}' -X github.com/hashicorp/vault/version.BuildDate=${BUILD_DATE}" \
```

Comment:
I think I'd prefer to set both the previous link-time variables but also the new ones (here, and below), if we aren't going to delete `sdk/version` quite yet.

Reply:
I've decided I favour the drop-sdk-version approach.

Reply:
:+1: